semtypes/src/main/java/io/ballerina/types/typeops/ListProj.java

```diff
@@ -82,24 +87,24 @@ static SemType listProjBdd(Context cx, SubtypeData k, Bdd b, Conjunction pos, Co
             return allOrNothing.isAll() ? listProjPath(cx, k, pos, neg) : NEVER;
         } else {
             BddNode bddNode = (BddNode) b;
-            return union(listProjBdd(cx, k, bddNode.left, and(bddNode.atom, pos), neg),
-                         union(listProjBdd(cx, k, bddNode.middle, pos, neg),
-                               listProjBdd(cx, k, bddNode.right, pos, and(bddNode.atom, neg))));
+            return union(listProjBdd(cx, k, bddNode.left(), and(bddNode.atom(), pos), neg),
```

Comment:
Need to update current method name to `listProjBddInnerVal`

Reply:
Also `listProj()` to `listProjInnerVal()`

Reply:
Fixed